ui/test/cypress/support/commands.d.ts

```diff
@@ -0,0 +1,38 @@
+/// <reference types="Cypress" />
```

Comment:
Do you think these commands could be useful for other devs too? If yes, we could add them into the Cypress AE

Reply:
The checks for vertical/horizontal position are specific to quasars positioning system for popups. I don't think that is very useful. The `isNotActionable` could be useful. When calling .click() on a covered element it will normally fail the test if that element is covered. This command is the reverse of that, it will fail if the .click() succeeds.